src/views/About.vue

```diff
@@ -37,24 +62,38 @@ export default {
 @import "../_variables.scss";
 
 .about {
-  display: grid;
+  margin: auto;
   margin-top: 3rem;
-  grid-template-columns: 1fr 2fr;
+  display: flex;
+  flex-flow: column;
+  width: 90vw;
+}
 
-  .sidebar {
-    margin-right: 0.3rem;
-    width: 13rem;
-  }
+.tabs > div {
+  position: relative;
+  display: inline-block;
+  padding: .4em 1em 0;
+  margin-bottom: .3em;
+}
 
-  .about-content {
-    @include beautifyBorder(.3rem, $bg_2, $bg_1, $bg_2, $eve-sky, $sky, $dark-blue,
-    0/ 0.8em 0.8em)
-    margin-left: 0rem;
-    margin-right: 1.3rem;
-    color: $black;
-    h2, h3, b {
-      color: $dark-blue;
-    }
-  }
+.tabs > div::before {
+  content: '';
+  position: absolute;
+  top: 0; right: -13px; bottom: 0; left: -13px;
+  z-index: -1;
+  background: #ccc;
+  background-image: linear-gradient(
+                      hsla(0,0%,100%,.6),
+                      hsla(0,0%,100%,0));
+  border: 1px solid rgba(0,0,0,.4);
+  border-bottom: none;
+  border-radius: 0 .15em white inset;
+  transform: scaleY(1.3) perspective(.5em) rotateX(5deg);
+  transform-origin: bottom;
+  height: 2rem;
```

Comment:
The trapezoidal tabs are not really there. Pseudo-elements to the rescue.

Reply:
Wow your CSS skillz, you must have beeen reading that textbook you showed me. It's like magic that they appear.